clientv3/balancer.go

```diff
@@ -64,6 +64,23 @@ func newSimpleBalancer(eps []string) *simpleBalancer {
 	return sb
 }
 
+func (b *simpleBalancer) update(m map[string]struct{}) {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+
+	addrs := make([]grpc.Address, 0, len(m))
+	for ep := range m {
+		addrs = append(addrs, grpc.Address{Addr: getHost(ep)})
+	}
+	b.addrs = addrs
+	b.notifyCh <- addrs
```

Comment:
@heyitsanthony So I am seeing gRPC is being notified of these new `addrs` via `Notify` and `lbWatchers`, but I am still seeing the same error in tests. Do you see anything else that I am doing wrong?